qiskit/providers/ibmq/experiment/experimentservice.py

```diff
@@ -335,6 +337,14 @@ def analysis_results(
                 ``analysis_results(quality=[('ge', 'Bad'), ('lt', 'Good')])``
                 will return all analysis results with a quality of ``Bad`` and
                 ``No Information``.
+            tags: Filter by tags assigned to experiments.
+            tags_operator: Logical operator to use when filtering by job tags. Valid
```

Comment:
```suggestion
            tags_operator: Logical operator to use when filtering by tags. Valid
```

This is probably also a copy/paste problem from the experiment tags, which probably also shouldn't say "job" tags since these aren't jobs.